hedera-node/hedera-token-service-impl/src/main/java/com/hedera/node/app/service/token/impl/handlers/transfer/AssociateTokenRecipientsStep.java

```diff
@@ -216,6 +201,47 @@ private TokenAssociation validateAndBuildAutoAssociation(
         }
     }
 
+    /**
+     * Dispatches a synthetic transaction to associate the token with the account. It will increment the usedAutoAssociations
+     * count on the account and set the token relation as automaticAssociation.
+     * This is done only if the unlimitedAutoAssociationsEnabled is enabled.
+     * @param token The token to associate with the account
+     * @param accountStore The account store
+     * @param tokenRelStore The token relation store
+     * @param context The context
+     * @param account The account to associate the token with
+     */
+    private void dispatchAutoAssociation(
+            final @NonNull Token token,
+            final @NonNull WritableAccountStore accountStore,
+            final @NonNull WritableTokenRelationStore tokenRelStore,
+            final @NonNull HandleContext context,
+            final @NonNull Account account) {
```

Comment:
Nit: Unless I am overlooking something, you do not need the `token` and `account`, but `tokenId` and `accountId` are sufficient.

Reply:
yes @netopyr . Will fix in the future PR